src/java/org/apache/ivy/plugins/report/XmlReportParser.java

```diff
@@ -193,16 +193,8 @@ public void startElement(String uri, String localName, String qName,
                         String branch = attributes.getValue("branch");
                         String revision = attributes.getValue("revision");
                         Map<String, String> extraAttributes = new HashMap<>();
```

Comment:
Is this Map used?

Reply:
It was, until I reused code for attributes extraction from `ExtendableItemHelper`.